### ISSUE TITLE
Acceptance Test: UC- 62

### ISSUE THREAD

**Issue by OmniaGo** [rThamb/soen390-schoolmap#103]
**UC-62, As a user, I want to set my preferred way to move inside the building. **

**Procedures**
Launch the app
Click on the hamburger button situated at the top left hand corner of the screen.
Select “Settings”
Choose the preferred way of moving around inside campuses
Go back to main menu
Select “New Route”
Set start to : H890
Set destination to: H40
Click on the button “Get Directions”

**Expected Output**
The path displayed to the user when they click on “Get Directions” will only contain directions based on their choice.

**Comment by mosaliza** [comment]
![Screenshot_20200409-130429_ConcordiaGo](https://user-images.githubusercontent.com/35632341/78921293-b0cf9800-7a62-11ea-8258-a316cec361eb.jpg)
Get direction does not work.
No hint for room names in buildings!

**Comment by JenniferAbouJaoudi** [comment]
Please test with the following:
Procedures
1- Launch the app
2- Click on the hamburger button situated at the top left hand corner of the screen.
3- Select “Settings”
4- Choose the preferred way of moving around inside campuses
5- Go back to main menu
6- Select “New Route”
7- Set start to : HB890
8- Set destination to: HB840
9- Click on the button “Get Directions”

**Comment by mosaliza** [comment]
Not working with the new instruction as well.

**Comment by MNezameddin** [comment]
Step 1: please select one or more options from the settings menu (if more than one is selected they are sorted by the following priority: 1. Elevators 2. Escalators 3.stairs)
![Screenshot_20200409-135929~2.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/1b32e418-7527-46fe-86d0-ebe6dff2cb3d)

Step 2: Go to new route and enter the classes mentioned (start: HB890 destination: HB922)
![Screenshot_20200409-140501.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/36f8cb79-0f8a-4bb1-aeec-018a40e5c534)

Step 3: click get directions and you'll be navigated to the first floor once again, and you'll be required to change to the 8th and 9th floor to view your path (in my case I chose escalators)
![Screenshot_20200409-140321.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/6442bdeb-8738-4746-8867-671ad81c3d98)

![Screenshot_20200409-140414.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/4f743985-6e26-465c-a6d6-dcfcb25c9ca6)

![Screenshot_20200409-140429.png](https://images.zenhubusercontent.com/5e1a89f389cf347b9596bb59/73ec425d-9ba8-4eca-b9c5-a06d49eced59)

Please keep in mind all screenshots were taken from my phone using the same APK file.

**Comment by mosaliza** [comment]
Did you test it on both ios or android phones?

**Comment by mosaliza** [comment]
Accepted(90%).
For sprint 4 fix it to work on android. I trust your screenshots.

**Comment by MNezameddin** [comment]
yes so all tests were done on an android phone (in my case it's a google pixel 3a XL). We ensured that the sdk worked at least on my phone, not 100% sure about IOS devices since I do not own one. Got all my screenshots directly from my phone, but if ever there are any issues, I can either film a demo on my phone or have a live demo with you if you'd like. 

Thank you.

**Comment by MNezameddin** [comment]
Please view the following video within the ZIP.

[UC-62.zip](https://github.com/rThamb/soen390-schoolmap/files/4473173/UC-62.zip)

Device Specs:

![Device_Specifications](https://user-images.githubusercontent.com/39063206/79186782-90983400-7de8-11ea-81d7-0d522e78bf7a.png)
